controllers/controller_core.js

```diff
@@ -0,0 +1,14 @@
+const selectTopics = require("../models/models_core")
+
+
+const getTopics = (req, res) => {
+  selectTopics()
+  .then((topics) =>
+  {
+    res.status(200).send({ topics })
+  }).catch((err) => {
+    console.log(err)
+  })
```

Comment:
Do any of the tests you have written currently require this catch block to be in your code?